### PR TITLE
added feature: profile-view-count #195

### PR DESCRIPTION
**Pull request by yyuvraj54**
->Now a person can get the profile view through this method -->  'get_Profile_Views_Count()'
->And if profile view counter is not present in profile it will return none 
->else i have done all sort of error handling and tested this code with different profiles (it is working perfectly)

### CODE REVIEW

**Comment by yyuvraj54** [comment]
My issue #195

File: src/scrape_up/github/users.py [hunk 1]
```diff
@@ -118,6 +118,30 @@ def repo_count(self):
...
+                        url =img_tag['data-canonical-src']
+                        response = requests.get(url)
+                        html_content = response.text
+                        soup = BeautifulSoup(html_content, 'html.parser')
+                        value_element = soup.find_all('g')
+                        for text_tag in value_element:
+                            if f'fill="#fff"' in str(text_tag): 
+                                valueLst=text_tag.find_all('text')
+                                return valueLst[-1].text
+        except:
```
Comment (nikhil25803) [inline src/scrape_up/github/users.py:142]: Needs few more changes @yyuvraj54 
+ Return the output with a data and message, you can check other methods for reference. 
+ Mention this method in the `documentation.md` file also.
+ Upload SS of the output for the validation.

**Review by nikhil25803 — CHANGES_REQUESTED** [review]

File: src/scrape_up/github/users.py (hunk 1 above)
Comment (yyuvraj54) [reply src/scrape_up/github/users.py:142]: ok working on it

Comment (yyuvraj54) [reply src/scrape_up/github/users.py:139]: When profile views are available
corrr


When profile views are not available
not

**Review by nikhil25803 — CHANGES_REQUESTED** [review]
Still not working for my profile. Instead of depending on an external URL, try finding an a trustworthy resource.
image

**Comment by nikhil25803** [comment]
Any update @yyuvraj54

**Comment by yyuvraj54** [comment]
There is no secure  way to get the views of a github account 

Screenshot 2023-06-15 at 8 38 28 PM

Screenshot 2023-06-15 at 8 32 21 PM

if the person have this above label in there readme.md file than my program is working like in my profile but person who does 
not have above label in readme file will return null 

Reason -> As we need some counter that is attached to profile (like above image) that takes count and from that i am fetching the values of profile views

**Comment by yyuvraj54** [comment]
I have raised new issue please check out this also->  #311

**Pull request closed without merging**